azure-pipelines.yml

```diff
@@ -558,7 +558,7 @@ jobs:
           - "chip_earlgrey_cw310_hyperdebug/rom.mmi"
           - "chip_earlgrey_cw310_hyperdebug/otp.mmi"
         gcpKeyFile: "gcpkey.json"
-        bucketURI: "gs://opentitan-bitstreams/master"
+        bucketURI: "gs://opentitan-bitstreams/earlgrey_es"
```

Comment:
We might need to update rules/scripts/bitstreams_workspace.py to look in the right place. There might be surprising effects for requesting the `latest`, since earlgrey_es and master bitstream caches would still go to the same local disk directory.

Reply:
It looks like I changed the default to be HEAD long ago anyway. `BITSTREAM=latest` never really makes sense, come to think of it, as it can cause you to bring in a bitstream that is newer than your current branch supports.

I think we could skip this.

Reply:
Agreed.